### PR TITLE
Fix gm-unit for skills above level 20

### PR DESCRIPTION
**Pull request by tatoyoda600**
Fixes DFHack/dfhack#4579
image

### CODE REVIEW

File: internal/gm-unit/editor_skills.lua [hunk 1]
```diff
@@ -24,7 +24,13 @@ function list_skills(unit, learned_only)
                     u_skill={rating=-1,experience=0}
                 end
                 local rating
-                if u_skill.rating >=0 then
+                if u_skill.rating >15 then
```
Comment (myk002) [inline internal/gm-unit/editor_skills.lua:27]: Please use `df.skill_rating.Legendary` instead of the magic number `15`

**Review by myk002 — COMMENTED** [review]
Thanks! please add a line to the `Misc Improvements` section under the "Future" version in `changelog.txt`

Also, please feel free to add yourself to https://github.com/DFHack/dfhack/blob/develop/docs/about/Authors.rst (in a separate PR)

File: internal/gm-unit/editor_skills.lua (hunk 1 above)
Comment (tatoyoda600) [reply internal/gm-unit/editor_skills.lua]: I've replaced the magic number and added an entry in `changelog.txt`.
I'm unsure if I made the changelog entry correctly though.

**Review by myk002 — APPROVED** [review]
looks great! Thanks!

**Pull request merged**